include/nleobs.h

```diff
@@ -45,6 +45,8 @@ typedef struct {
 #ifdef NLE_ALLOW_SEEDING
     unsigned long seeds[2]; /* core, disp */
     char reseed; /* boolean: use NetHack's anti-TAS reseed mechanism? */
+    int spawn_monsters; /* boolean: spawn monsters randomly after every step
```

Comment:
Did you mean for this to be in the #ifdef?

Reply:
Good catch. Forgot to remove this from the struct.